src/packages/aggregate-report/aggregate-report.test.ts

```diff
@@ -0,0 +1,27 @@
+import aggregateReport from './';
+import PerryReportInfo from './../../interfaces/PerryReportInfo';
+
+const reportInfo : PerryReportInfo = {
+  title: 'Testing',
+  description: 'Testing this aggregateReport',
+  screenshotUrl: 'http://urlscreenshot.com',
+};
+
+const expectReport = {
+    "clicks": null,
+    "cookies": "",
+    "description": "Testing this aggregateReport",
+    "errors": [],
+    "logs": null,
+    "notify": null,
+    "screenshotUrl":
+    "http://urlscreenshot.com",
+    "title": "Testing",
+    "warns": null
+}
+
+describe('Aggregate report', () => {
+  test('Testing aggregate report function', () => {
```

Comment:
I think we should be strict into behavior-driven development here, you should be using `it` instead of `test` here, it would look a little bit like this:

```js
describe('Aggregate Report', () => {
  it('should return an object that matches the expected report', () => {
    expect(aggregateReport(reportInfo)).toEqual(expectedReport);
  });
});
```